### PR TITLE
frontend changes

### PR DESCRIPTION
**Pull request by maefranco**
removed strange symbols in contact-us.html
corrected css classes in forms in model.py
included fonts folder

### CODE REVIEW

**Comment by jthreatt4** [comment]
@maefranco i can't commit this. There are no fixtures for the about page, workshop-index or blog-index. Also the changes to the form app break the css for the fields. The template for the forms needs the html and css for the grid framework to set widths on form fields. The classes i provided are needed for the javascript widgets on the forms to work. Also the media files folder shouldn't be included in the repo.

**Comment by maefranco** [comment]
I'm working on the fixtures for the about page, workshop-index and blog-index. Having the CSS classes in the create radio, checkboxes, and dropdown fields caused the radio, dropdown, and check boxes to be superimposed over text fields, which is why I removed those classes from those specific fields. I'll send you screenshots of how a created form looked before and after I removed those CSS classes.
I also included the media folder in gitignore
forms_before
form_after

**Comment by jthreatt4** [comment]
I see what your saying on the checkboxes and radio buttons, but there aren't any styles for them in the template?

**Comment by maefranco** [comment]
There is styling for them in the template, but for some reason, including the CSS classes (.form-control, etc) in the def create radio and checkbox fields in forms/models.py created text boxes as well as radio/check buttons. I'm not sure why that is, but removing the classes from those specific fields solved the problem. Hope that answers your question.

**Comment by maefranco** [comment]
Julian, I didn't resolve the conflict regarding import statements in home/models.py prior to creating the pull request. Do I need to create a new request?

**Comment by jthreatt4** [comment]
no anything you push to an open pull request updates the pull request.

File: home/models.py [hunk 1]
```diff
@@ -4,10 +4,16 @@
...
+<<<<<<< HEAD
+from wagtail.wagtailimages.blocks import ImageChooserBlock
+from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+from wagtail.wagtailcore.fields import RichTextField, StreamField
+=======
 from wagtail.wagtaildocs.blocks import DocumentChooserBlock
 from wagtail.wagtailimages.blocks import ImageChooserBlock
 from wagtail.wagtailcore.fields import RichTextField, StreamField
 from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+>>>>>>> 4ea952ce95eede0524fa8421b399824fefd630b8
```
Comment (jthreatt4) [inline home/models.py]: this merge conflict wasn't resolved

**Pull request merged**